src/support/string.h

```diff
@@ -96,7 +96,10 @@ inline bool wildcardMatch(const std::string& pattern,
       return false;
     }
     if (pattern[i] == '*') {
-      return true;
+      return wildcardMatch(pattern.substr(i + 1, std::string::npos),
+                           value.substr(i, std::string::npos)) ||
+             wildcardMatch(pattern.substr(i, std::string::npos),
+                           value.substr(i + 1, std::string::npos));
```

Comment:
the `npos` parameters are not needed (they are the default value)

Reply:
Done.